src/windows/main_window.py

```diff
@@ -1332,6 +1335,20 @@ def actionNextMarker_trigger(self, event):
             get_app().window.refreshFrameSignal.emit()
             get_app().window.propertyTableView.select_frame(frame_to_seek)
 
+    def addKeyframesToMarkers(self, data):
+        for property in data :
+            try :
+                for point in data[property]["Points"] :
+                    keyframe=(point["co"]["X"]-1)/fps_float - data["start"] + data["position"]
+                    if keyframe > start and keyframe < stop :
+                        all_marker_positions.append(keyframe)
+            except TypeError:
+                log.info("%s : %s : not itterable", property, data[property])
+                pass
+            except KeyError:
+                log.info("%s : %s : has no points", property, data[property])
+                pass
```

Comment:
I don't think there's any reason to _log_ either of these conditions. You won't actually get a `TypeError` for non-iterables, you'll hit it even earlier than that since non-keyframable properties like `data["duration"]` don't have a `Points` member, so the error will be that they're non-_subscriptable_. But that's expected and not an extraordinary condition.

Any properties without any keyframe points similarly don't need to be logged; that information is of no use to the user, or even to the devs in the context of the markers list. So the exceptions can simply be ignored.
```suggestion
            except (TypeError, KeyError):
                pass
```